lib/mintacoin/accounts/crypto/stellar.ex

```diff
@@ -16,28 +16,52 @@ defmodule Mintacoin.Accounts.Stellar do
 
   @impl true
   def create_account(_opts) do
-    {fund_public_key, _fund_private_key} = fund_keypair = fund_key_pair_from_system()
+    fund_info = fund_information()
     {public_key, secret_key} = KeyPair.random()
 
-    source_account = TxBuild.Account.new(fund_public_key)
-    {:ok, seq_num} = Horizon.Accounts.fetch_next_sequence_number(fund_public_key)
-    sequence_number = TxBuild.SequenceNumber.new(seq_num)
-
     operation =
-      TxBuild.CreateAccount.new(
-        destination: public_key,
-        starting_balance: 1
-      )
+      public_key
+      |> build_create_account_operation()
+
+    envelope = build_envelope(fund_info.fund_public_key, operation, fund_info.fund_signature)
+
+    envelope
+    |> execute_transaction(public_key, secret_key)
+  end
 
-    signature = TxBuild.Signature.new(fund_keypair)
+  defp fund_information do
```

Comment:
@Odraxs Let's rename this function like this:
```suggestion
  defp funder_account_information do
```